cmd/gateway-main.go

```diff
@@ -139,12 +141,14 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 
 	// Validate if we have access, secret set through environment.
 	if !globalIsEnvCreds {
-		errorIf(fmt.Errorf("Access and secret keys not set"), "Access and Secret keys should be set through ENVs for backend [%s]", gatewayName)
+		reqInfo := (&logger.ReqInfo{}).AppendTags("gatewayName", gatewayName)
+		contxt := logger.SetReqInfo(context.Background(), reqInfo)
+		logger.LogIf(contxt, fmt.Errorf("Access and Secret keys should be set through ENVs for backend"))
```

Comment:
No need to use `fmt.Error`. There is nothing to be formatted.   We can use `errors.New`.